tools/biolatency.py

```diff
@@ -18,12 +18,13 @@
 
 # arguments
 examples = """examples:
-    ./biolatency            # summarize block I/O latency as a histogram
-    ./biolatency 1 10       # print 1 second summaries, 10 times
-    ./biolatency -mT 1      # 1s summaries, milliseconds, and timestamps
-    ./biolatency -Q         # include OS queued time in I/O time
-    ./biolatency -D         # show each disk device separately
-    ./biolatency -F         # show I/O flags separately
+    ./biolatency                    # summarize block I/O latency as a histogram
+    ./biolatency 1 10               # print 1 second summaries, 10 times
+    ./biolatency -mT 1              # 1s summaries, milliseconds, and timestamps
+    ./biolatency -Q                 # include OS queued time in I/O time
+    ./biolatency -D                 # show each disk device separately
+    ./biolatency -F                 # show I/O flags separately
+    ./biolatency -j                 # print a dictionary
```

Comment:
Suppose we display results with -D
```
./biolatency.py -D
Tracing block device I/O... Hit Ctrl-C to end.
^C

disk = b'vda'
     usecs               : count     distribution
         0 -> 1          : 0        |                                        |
         2 -> 3          : 0        |                                        |
         4 -> 7          : 0        |                                        |
         8 -> 15         : 0        |                                        |
        16 -> 31         : 0        |                                        |
        32 -> 63         : 0        |                                        |
        64 -> 127        : 3        |******                                  |
       128 -> 255        : 4        |********                                |
       256 -> 511        : 18       |****************************************|
       512 -> 1023       : 10       |**********************                  |
      1024 -> 2047       : 3        |******                                  |
```

-D -j is not supported:
```
./biolatency.py -j -D
^C
Traceback (most recent call last):
  File "./biolatency.py", line 240, in <module>
    print_json_hist(dist)
  File "./biolatency.py", line 208, in print_json_hist
    max_nonzero_idx = k.value
AttributeError: 'disk_key' object has no attribute 'value'
```

I think we should support -j with all existing other flags. Could you check to make sure this is implemented properly?

Reply:
Fix pushed.